cuda_bindings/pixi.toml

```diff
@@ -0,0 +1,130 @@
+[workspace.build-variants]
+python = ["3.10.*", "3.11.*", "3.12.*", "3.13.*", "3.14.*"]
+
+[dependencies]
+cuda-bindings = { path = "." }
+pip = ">=25.3,<26"  # installed to the workspace for convenience
+
+[feature.test.dependencies]
+pytest = ">=6.2.4"
+pytest-benchmark = ">=3.4.1"
+pyglet = ">=2.1.9"
+numpy = "*"
+
+[feature.cython-tests.dependencies]
+cython = "*"             # for tests that exercise APIs from cython
+setuptools = "*"         # for distutils
+gxx = "*"                # to compile the generated code
+# These are necessary because running the Cython tests requires compiling
+# *after* the package is built, and the Cython tests depend on CUDA headers
+# transitively (through cuda-bindings)
+cuda-cudart-dev = "*"
+cuda-profiler-api = "*"
+
+[feature.cython-tests.target.linux-64.dependencies]
+cuda-crt-dev_linux-64 = "*"
+
+[feature.cython-tests.target.linux-aarch64.dependencies]
+cuda-crt-dev_linux-aarch64 = "*"
+
+[feature.cython-tests.target.win-64.dependencies]
+cuda-crt-dev_win-64 = "*"
+
+# For finding headers when building the Cython tests
+[feature.cython-tests.target.linux-64.activation.env]
+CUDA_HOME = "$CONDA_PREFIX/targets/x86_64-linux"
+
+[feature.cython-tests.target.linux-aarch64.activation.env]
+CUDA_HOME = "$CONDA_PREFIX/targets/sbsa-linux"
+
+[feature.cython-tests.target.win-64.activation.env]
+CUDA_HOME = '%CONDA_PREFIX%\Library'
+
+[feature.cu13.system-requirements]
+cuda = "13"
+
+[feature.cu13.dependencies]
+# TODO: make cuda-version a build-variant
+cuda-version = "13.*"
+
+[environments]
+cu13 = { features = ["cu13", "test", "cython-tests"], solve-group = "cu13" }
+
+# TODO: check if these can be extracted from pyproject.toml
+[package]
+name = "cuda-bindings"
+version = "13.0.2"
+
+[package.build]
+backend = { name = "pixi-build-python", version = "*" }
+
+[package.build.config]
+env = { CUDA_PYTHON_PARSER_CACHING = "1", CUDA_PYTHON_PARALLEL_LEVEL = "$(nproc)" }
+noarch = false
+compilers = ["c", "cxx"]
+
+[package.build.target.linux-64.config.env]
+CUDA_HOME = "$PREFIX/targets/x86_64-linux"
+
+[package.build.target.linux-aarch64.config.env]
+CUDA_HOME = "$PREFIX/targets/sbsa-linux"
+
+[package.build.target.win-64.config.env]
+CUDA_HOME = '%PREFIX%\Library'
+
+# TODO: revisit this once prefix-dev/pixi#4906 is resolved
+# [package.build-dependencies]
+# ccache = "*"
+
+[package.host-dependencies]
+python = "*"
+pip = "*"
+setuptools = ">=77"
+cython = ">=3.1,<3.2"
+pyclibrary = ">=0.1.7"
+cuda-cudart-static = "*"
+cuda-nvrtc-dev = "*"
+cuda-profiler-api = "*"
+cuda-nvvm = "*"
+
+[package.target.linux-64.host-dependencies]
+cuda-crt-dev_linux-64 = "*"
+libcufile-dev = "*"
+
+[package.target.linux-aarch64.host-dependencies]
+cuda-crt-dev_linux-aarch64 = "*"
+libcufile-dev = "*"
+
+[package.target.win-64.host-dependencies]
+cuda-crt-dev_win-64 = "*"
+
+[package.run-dependencies]
+python = "*"
+cuda-pathfinder = ">=1.1,<2"
+cuda-bindings = { path = "." }
+libnvjitlink = "*"
+cuda-nvrtc = "*"
+cuda-nvvm = "*"
```

Comment:
I aligned pixi.toml with our feedstock recipe. All run-time dependencies are moved to under `[package.run-dependencies]` except for `cuda-bindings` itself, which cannot be placed here if we want a source build.